aws-ec2-instance-block-device-unencrypted.yaml

```diff
@@ -0,0 +1,60 @@
+rules:
+- id: aws-ec2-instance-block-device-unencrypted
+  pattern-either:
+  - patterns:
+    - pattern: |
+        resource "aws_instance" $ANYTHING {
+          ...
+        }
+    - pattern-not-inside: |
+        resource "aws_instance" $ANYTHING {
+          ...
+          root_block_device {
+            ...
+            encrypted = true
+            ...
+          }
+          ...
+        }
+  - patterns:
+    - pattern-inside: |
+        resource "aws_instance" $ANYTHING {
+          ...
+        }
+    - pattern: |
+        ebs_block_device {
+          ...
+        }
+    - pattern-not: |
+        ebs_block_device {
+          ...
+          encrypted = true
+          ...
+        }
+  message: The AWS EC2 block device is unencrypted. The block device could be read
```

Comment:
My rule **does** use  the yaml operator `>-`:

<img width="594" alt="Screenshot 2023-02-23 at 3 34 05 PM" src="https://user-images.githubusercontent.com/5559014/221023791-2c17fa51-0fda-42fe-96d4-bbf0815ae440.png">

I don't know why that disappears when I publish it from the semgrep app?